qiskit/providers/aer/noise/noise_model.py

```diff
@@ -1034,3 +1066,15 @@ def _nonlocal_quantum_errors_equal(self, other):
                     if iinner_dict1[iinner_key] != iinner_dict2[iinner_key]:
                         return False
         return True
+
+    def _pass_manager(self) -> Optional[PassManager]:
+        """
+        Return the pass manager that add custom noises defined as noise passes
+        (stored in the _custom_noise_passes field). Note that the pass manager
+        does not include passes to add other noises (stored in the different field).
+        """
+        passes = []
+        passes.extend(self._custom_noise_passes)
+        if len(passes) > 0:
+            return PassManager(passes)
+        return None
```

Comment:
The fix was included in terra 0.19 that an empty pass manager just returns the input circuits without dag conversions.